dvc/scm/git.py

```diff
@@ -381,6 +381,15 @@ def remind_to_track(self):
             "\tgit add {files}".format(files=files)
         )
 
+    def track_changed_files(self):
+        if not self.files_to_track:
+            return
+
+        self.add(self.files_to_track)
+        self.reset_tracked_files()
+
+        logger.info("\nChanges staged to git.")
```

Comment:
Wonder if this message might be confusing to people, that might think that their data is now tracked by git (yeah, we do actually encounter such cases from time to time :slightly_smiling_face: ). Maybe we could just not print anything here? Seems like an expected behavior, considering an explicit config option.

Reply:
Sure, I can't think of a better solution to that particular problem!